src/renderers/shared/fiber/ReactFiber.js

```diff
@@ -144,8 +145,12 @@ export type Fiber = {
 
 };
 
+export type FiberDev = Fiber & {
```

Comment:
Yea. I think we'll want to change the constructor logic there at some point to pass everything that can be non-nullable to satisfy the use case in #8545 but we probably need to trick Flow to follow it regardless. Perhaps we can use `declare __DEV__ : true;` somewhere to trick Flow into always taking that path (although that might trigger unused paths and miss errors in those paths).